ios/Classes/PurchasesFlutterPlugin.m

```diff
@@ -479,10 +492,23 @@ - (void)closeWithResult:(FlutterResult)result {
 #pragma mark Delegate Methods
 
 - (void)purchases:(RCPurchases *)purchases didReceiveUpdatedPurchaserInfo:(RCPurchaserInfo *)purchaserInfo {
-    [self.channel invokeMethod:RNPurchasesPurchaserInfoUpdatedEvent
+    [self.channel invokeMethod:PurchasesPurchaserInfoUpdatedEvent
                      arguments:purchaserInfo.dictionary];
 }
 
+-(void) purchases:(RCPurchases *)purchases shouldPurchasePromoProduct:(SKProduct *)product defermentBlock:(RCDeferredPromotionalPurchaseBlock)makeDeferredPurchase {
```

Comment:
nits: 
- extra space between `(void)` and `purchases:`
- line breaks between params